roles/openshift_node/handlers/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
-- name: restart openshift-node
-  service: name=openshift-node state=restarted
+- name: restart node
+  service: name={{ product_type }}-node state=restarted
```

Comment:
The way I'm interpreting things is that we'll have two sets of packages and two sets of services.
origin-{master,node} and atomic-openshift-{master,node} services
origin{,-master,-node,-sdn-ovs} and atomic-openshift{,-master,-node,-sdn-ovs} packages.